tmt/steps/__init__.py

```diff
@@ -418,9 +418,15 @@ def _normalize_data(
     def _export(self, *, keys: Optional[List[str]] = None) -> tmt.export._RawExportedInstance:
         # TODO: one day, this should recurse down into each materialized plugin,
         # to give them chance to affect the export of their data.
-        # TODO: `key` isincorrect, `option` is the correct one, and this will happen to fix
-        # https://github.com/teemtee/tmt/issues/2054
-        return cast(tmt.export._RawExportedInstance, self._raw_data)
+
+        def _export_datum(raw_datum: _RawStepData) -> _RawStepData:
+            return cast(
+                _RawStepData,
+                {key_to_option(key): value for key, value in raw_datum.items()})
+
+        return cast(
+            tmt.export._RawExportedInstance,
+            [_export_datum(raw_datum) for raw_datum in self._raw_data])
```

Comment:
`datum`? Shouldn't it be `date` or more likely `data`?

Reply:
"datum" as a singular form of "data". Definitely not representing a calendar date. Rarely used, I guess, yet still correct, AFAICT. Since `raw_data` and `data` in general, in the case of steps, means "a collection of step data packages", I wanted a term representing a single entry from this list, a single package for one particular phase of the step.

Reply:
Ack. And it's consistently used in this way throughout the code already. Marking as resolved.